src/features/configure/vault/polygon_pools.js

```diff
@@ -1,4 +1,85 @@
+    name: 'maUSDC-USDC LP',
+    token: 'maUSDC-USDC QLP',
+    tokenDescription: 'QuickSwap',
+    tokenAddress: '0x7295304b10740BA8e037826787d3e9386FD99925',
+    tokenDecimals: 18,
+    tokenDescriptionUrl: '#',
+    earnedToken: 'mooQuickmaUSDC-USDC',
+    earnedTokenAddress: '0x8c2d54BA94f4638f1bb91f623F378B66d6023324',
+    earnContractAddress: '0x8c2d54BA94f4638f1bb91f623F378B66d6023324',
+    pricePerFullShare: 1,
+    tvl: 0,
+    oracle: 'lps',
+    oracleId: 'quick-mausdc-usdc',
+    oraclePrice: 0,
+    depositsPaused: false,
+    status: 'active',
+    platform: 'QuickSwap',
+    assets: ['maUSDC', 'USDC'],
+    callFee: 0.5,
+    addLiquidityUrl:
+      'https://quickswap.exchange/#/add/0x9719d867A500Ef117cC201206B8ab51e794d3F82/0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174',
+    buyTokenUrl:
+      'https://quickswap.exchange/#/swap?inputCurrency=0x2791bca1f2de4661ed88a30c99a7a9449aa84174&outputCurrency=0x9719d867a500ef117cc201206b8ab51e794d3f82',
+  }, 
+  {
+    id: 'quick-aave-eth',
+    logo: 'polygon/AAVE-ETH.png',
+    name: 'AAVE-ETH LP',
+    token: 'AAVE-ETH QLP',
+    tokenDescription: 'QuickSwap',
+    tokenAddress: '0x90bc3E68Ba8393a3Bf2D79309365089975341a43',
+    tokenDecimals: 18,
+    tokenDescriptionUrl: '#',
+    earnedToken: 'mooQuickAAVE-ETH',
+    earnedTokenAddress: '0x752948B4493d2Ee09c95F944A76005aEBF410087',
+    earnContractAddress: '0x752948B4493d2Ee09c95F944A76005aEBF410087',
+    pricePerFullShare: 1,
+    tvl: 0,
+    oracle: 'lps',
+    oracleId: 'quick-aave-eth',
+    oraclePrice: 0,
+    depositsPaused: false,
+    status: 'active',
+    platform: 'QuickSwap',
+    assets: ['AAVE', 'ETH'],
+    callFee: 0.5,
+    addLiquidityUrl:
+      'https://quickswap.exchange/#/add/0xD6DF932A45C0f255f85145f286eA0b292B21C90B/0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619',
+    buyTokenUrl:
+      'https://quickswap.exchange/#/swap?inputCurrency=0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39&outputCurrency=ETH',
+  }, 
+  {
+    id: 'quick-link-eth',
+    logo: 'polygon/LINK-ETH.png',
+    name: 'LINK-ETH LP',
+    token: 'LINK-ETH QLP',
+    tokenDescription: 'QuickSwap',
+    tokenAddress: '0x5cA6CA6c3709E1E6CFe74a50Cf6B2B6BA2Dadd67',
+    tokenDecimals: 18,
+    tokenDescriptionUrl: '#',
+    earnedToken: 'mooQuickLINK-ETH',
+    earnedTokenAddress: '0xaB4105375FbE5F502B0da986F46ADf7a21762e52',
+    earnContractAddress: '0xaB4105375FbE5F502B0da986F46ADf7a21762e52',
+    pricePerFullShare: 1,
+    tvl: 0,
+    oracle: 'lps',
+    oracleId: 'quick-link-eth',
+    oraclePrice: 0,
+    depositsPaused: false,
+    status: 'active',
+    platform: 'QuickSwap',
+    assets: ['LINK', 'ETH'],
+    callFee: 0.5,
+    addLiquidityUrl:
+      'https://quickswap.exchange/#/add/0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39/0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619',
+    buyTokenUrl:
+      'https://quickswap.exchange/#/swap?inputCurrency=0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39&outputCurrency=ETH',

```

Comment:
https://quickswap.exchange/#/swap?inputCurrency=0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39&outputCurrency=0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619

Here as well (ETH gets mapped to Matic, most likely since Uni fork and Matic is native token)